zubhub_frontend/zubhub/src/views/create_team/step3/step3.styles.js

```diff
@@ -23,7 +23,7 @@ export const step3Style = theme => ({
     },
     overLay:{height:'100%', width:'100%',top:0, position:'absolute', backgroundColor:'transparent', zIndex:1},
     blurOverLay:{
-        borderRadius: '6.93px',
+        borderRadius: '15px',
```

Comment:
Can you please attach UI differences `before  and after` the changes in the summary above. It's easier for reviewers. 

We actually have to run this locally and test these changes to confirm if it is actually what you say it is. So attaching some screenshots is crucial more so for such UI changes